dotnet/Gherkin/Parser.cs

```diff
@@ -75,8 +75,20 @@ public enum RuleType
     [System.Runtime.CompilerServices.CompilerGeneratedAttribute()]
     public partial class Parser<T>
     {
+        public Parser()
+            : this(new AstBuilder<T>())
+        {
+        }
+
+        public Parser(AstBuilder<T> astBuilder)
+        {
+            this.AstBuilder = astBuilder;
+        }
+
         public bool StopAtFirstError { get; set;}
 
+        public AstBuilder<T> AstBuilder { get; }

```

Comment:
Isn't `readonly` the keyword in C# to say that this instance variable cannot change/is constant?